packages/opentelemetry-context-base/.eslintignore

```diff
@@ -0,0 +1,2 @@
+build
```

Comment:
Is it possible to add this file in top level packages instead within each package?

Reply:
No, it's not possible it didn't find the .eslintignore file at the top level